stories/Popover/Popover.stories.jsx

```diff
@@ -0,0 +1,42 @@
+import Popover from '../../components/Popover';
+import { Row, Col } from '../../components';
+
+const Template = args => {
+  const { ...rest } = args;
+  return (
+    <Row
+      style={{
+        padding: '44px 0',
+        display: 'flex',
+        flexDirection: 'column',
+        justifyContent: 'center',
+        alignItems: 'center',
+      }}
+    >
+      <Col>
+        <Popover trigger="TriggerText" {...rest}>
+          childrenText
+        </Popover>
+      </Col>
+    </Row>
+  );
+};
+
+export const Default = Template.bind({});
+Default.args = {};
```

Comment:
```suggestion
```
When an example has no parameters it is not necessary to include an "args" object